flow/envs/loop/loop_merges.py

```diff
@@ -22,7 +22,7 @@
 }
 
 
-class TwoLoopsMergeEnv(Env):
+class TwoLoopsMergePOEnv(Env):
     """Environment for training cooperative merging behavior in a closed loop
```

Comment:
modify description to indicate that it is partially observed?

Reply:
done